jdaviz/configs/default/plugins/collapse/tests/test_collapse.py

```diff
@@ -36,3 +38,28 @@ def test_linking_after_collapse(cubeviz_helper, spectral_cube_wcs):
     # Pixel Axis 0 [y] from plugin.pixel_component_ids[0]
     assert dc.external_links[1].cids1[0] == dc[0].pixel_component_ids[1]
     assert dc.external_links[1].cids2[0] == dc[-1].pixel_component_ids[0]
+
+
+def test_save_collapsed_to_fits(cubeviz_helper, spectral_cube_wcs):
+
+    cubeviz_helper.load_data(Spectrum1D(flux=np.ones((3, 4, 5)) * u.nJy, wcs=spectral_cube_wcs))
+
+    collapse_plugin = cubeviz_helper.plugins['Collapse']
+
+    # make sure export enabled is true, and that before the collapse function
+    # is run `collapsed_spec_available` is correctly set to False
+    assert collapse_plugin._obj.export_enabled
+    assert collapse_plugin._obj.collapsed_spec_available is False
+
+    # run collapse function, and make sure `collapsed_spec_available` was set to True
+    collapse_plugin._obj.vue_collapse()
+    assert collapse_plugin._obj.collapsed_spec_available
+
+    # save output file with default name, make sure it exists
+    collapse_plugin._obj.vue_save_as_fits()
```

Comment:
Should make sure that it is writing out the file to `tmp_path`, otherwise our source tree will get poluted by random output files during local testing, and someone might accidentally check it into version control.

Reply:
done